app/views/dashboards/show.html.slim

```diff
@@ -0,0 +1 @@
+| Dashboard Page
```

Comment:
Can we add some info about current signed user here for make sure we signed in under a correct Customer?